unishox2.c

```diff
@@ -129,11 +129,9 @@ void init_coder() {
   for (int i = 0; i < 3; i++) {
     for (int j = 0; j < 28; j++) {
       byte c = usx_sets[i][j];
-      if (c != 0 && c > 32) {
```

Comment:
The `c > 32` is required.  `c != 0` is redundant, but it indicates 0 values are not to be processed.

Reply:
Fixed